apisix/stream/xrpc/runner.lua

```diff
@@ -70,9 +74,49 @@ local function put_req_ctx(session, ctx)
 end
 
 
+local function filter_logger(ctx, logger)
+    if not logger or not logger.filter or #logger.filter == 0 then
```

Comment:
The logger can't be nil, and we should return true when filter is missing?

Reply:
update